### PR TITLE
Career page

### PR DESCRIPTION
**Pull request by bkase**
Career page of website modulo the following:

* [x] Images of people at the company
* [x] The proper font for the heading
* [x] Updated copy on the actual career site

site

Co-authored-by: Evan Shapiro <es92@users.noreply.github.com>

### CODE REVIEW

File: src/app/website/src/careers.ml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                      ; span [Style.just "i"] [text "at least"]
+                      ; text "14 days&nbsp;annually" ] ]
+                ; Benefit.create "Meals"
+                    [[text "Healthy snacks and provided lunch twice a week"]]
+                ; Benefit.create "Other"
+                    [ [text "Parental leave"]
+                    ; [text "Commuting benefits"]
+                    ; [text "Bike-friendly culture"]
+                    ; [ text
+                          "Take up to 1 day (8 hours) of paid time off per \
```
Comment (johnwu93) [inline src/app/website/src/careers.ml]: The volunteer text does not fit on one line and it actually goes to the next line. I thought the next line was another different benefit but it wasn't. Maybe we should have bullets to distinguish what is a benefit and what is a continuation of the next statement. 

The easiest thing to do is probably to make the statement shorter.

File: src/app/website/jobs/frontend-engineer.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Senior Frontend Engineer
```
[the remark this answers is not in the document]
Comment (bkase) [reply src/app/website/jobs/frontend-engineer.markdown]: I'll make sure this renders to `frontend-engineer.html` and `senior-frontend-engineer.html` so we don't get dead links if recruiters have used either one

File: src/app/website/src/careers.ml [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+let content =
+  let open Html_concise in
+  let content =
+    div
+      [Style.just "mw960 pv3 center ph3 ibmplex oceanblack"]
+      [ Title.create "Work with us!"
+      ; Image_grid.create
+          (Non_empty_list.init
+             (Image.draw "/static/img/careers/group-outside.jpg" `Free)
+             [ Image.draw "/static/img/careers/group-in-house.jpg" `Free
```
[the remark this answers is not in the document]
Comment (bkase) [reply src/app/website/src/careers.ml]: Not easily -- I can fix this in a future update when we replace this image with a better one.

File: src/app/website/jobs/frontend-engineer.markdown (hunk 2 above)
[the remark this answers is not in the document]
Comment (bkase) [reply src/app/website/jobs/frontend-engineer.markdown]: for now I'll keep both contents the same, but have two different files

**Pull request merged**